src/core/control/XournalMain.cpp

```diff
@@ -68,9 +68,9 @@ auto migrateSettings() -> MigrateResult;
 void checkForErrorlog();
 void checkForEmergencySave(Control* control);
 
-auto exportPdf(const char* input, const char* output, const char* range, ExportBackgroundType exportBackground,
+[[maybe_unused]] auto exportPdf(const char* input, const char* output, const char* range, ExportBackgroundType exportBackground,
                bool progressiveMode) -> int;
-auto exportImg(const char* input, const char* output, const char* range, int pngDpi, int pngWidth, int pngHeight,
+[[maybe_unused]] auto exportImg(const char* input, const char* output, const char* range, int pngDpi, int pngWidth, int pngHeight,
```

Comment:
Same as with member pointers: here only the forward declaration is unused (because the functions are used after their actual declarations). You can remove the forward declarations.